### ISSUE TITLE
JAWS does not announce the message bar

### ISSUE THREAD

**Issue by drotara** [microsoft/fluentui#17187]
### Environment Information

- **Package version(s)**: 7.153.2 +
- **Browser and OS versions**: Edge 88.0.705.50

### Describe the issue:
When focus is auto set on a page to something else (Lets say a backbutton) but a message bar is displayed Jaws does not read out the message bar. It only reads out the button that is focused on. 

### Please provide a reproduction of the issue in a codepen:
https://codepen.io/adrotar/details/VwmQvOx
-> Using the "Details" view and "Rerun" button it will set focus to the button and observe. 

#### Actual behavior:
JAWS just reads out the button that is selected instead of also reading the message bar that is on the page.

#### Expected behavior:
It should read out the message bar information and then read out the selected focused item. 

### Documentation describing expected behavior
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7b733120d0-56c5-479c-889f-fee36ea08ee0%7d&cid=5b1a2dd4-9eb1-401c-9972-12eaa6e2bcfc

**Comment by gouttierre** [comment]
@drotara - thanks for submitting this issue with us - I saw the behavior you mentioned on the ticket.

@khmakoto - Would you be able to confirm if this is a regression that the message to the left does not get cited, or if this behavior is an issue?

cc @smhigley

**Comment by Dalimil** [comment]
@gouttierre @khmakoto @joschect We are also running into this issue. The current Message Bar is not accessible, because narrator doesn't read its contents, the focus jumps directly to MessageBar link or button, so it doesn't read the actual content of the Message Bar.

Should the MessageBar container be a tab stop as a whole?

**Comment by smhigley** [comment]
@drotara unfortunately, that's an expected result of moving focus at the same time as firing a live region event. Different screen readers will resolve live region conflicts (whether between competing live regions, or live regions vs. user actions) differently. That's why it's best practice to not update a live region at the same time as a programmatic focus change, because they are guaranteed to conflict.

@Dalimil could you elaborate on your use case? The MessageBar shouldn't be stealing focus when it appears; if it does, that seems like a much bigger accessibility issue than the live region conflict.
